### PR TITLE
retain (truncated) command descriptions, add getting started

### PR DESCRIPTION
**Pull request by ashleymichal**
also added whitespace for markdown list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,16 +31,91 @@ If you don't have `cargo` or `npm` installed, you will need to follow these [add
 
 For information regarding updating Wrangler, click [here](https://workers.cloudflare.com/docs/quickstart/updating-the-cli/).
```
Comment (victoriabernard92) [inline README.md]:
```suggestion
```

Comment (ashleymichal) [reply README.md]: you left this in!

File: README.md [hunk 2]
```diff
@@ -31,16 +31,91 @@ If you don't have `cargo` or `npm` installed, you will need to follow these [add
...
 
+## Getting Started
+
+Once you have installed Wrangler, spinning up and deploying your first Worker is easy!
+
+```console
+$ wrangler generate my-worker
+$ cd my-worker
+# update your wrangler.toml with your Cloudflare Account ID
+$ wrangler config # => enter your api key and email
```
Comment (victoriabernard92) [inline README.md]:
```suggestion
$ wrangler config 
```
This is changing so I say just remove and let the prompt of the command instruct them

File: README.md [hunk 3]
```diff
@@ -31,16 +31,91 @@ If you don't have `cargo` or `npm` installed, you will need to follow these [add
...
+$ wrangler publish
+```
+
+## 🎙️ Top Level Commands
+
+- ### 👯 `generate`
```
Comment (victoriabernard92) [inline README.md]:
```suggestion
### 👯 `generate`
```
These bullets mess up the markdown rendering

File: README.md [hunk 4]
```diff
@@ -31,16 +31,91 @@ If you don't have `cargo` or `npm` installed, you will need to follow these [add
...
+
+  ```bash
+  wrangler generate <name> <template> --type=["webpack", "javascript", "rust"]
+  ```
+
+  All of the arguments and flags to this command are optional:
+  
+  - `name`: defaults to `worker`
+  - `template`: defaults to the [`https://github.com/cloudflare/worker-template`](https://github.com/cloudflare/worker-template)
+  - `type`: defaults to ["webpack"](https://github.com/cloudflare/wrangler/blob/master/docs/content/webpack.md)
```
Comment (victoriabernard92) [inline README.md]: Find and replace all links should be:

https://github.com/cloudflare/wrangler/blob/master/docs/content => developer.cloudflare.com


```
https://github.com/cloudflare/wrangler/blob/master/docs/content/webpack.md => https://developers.cloudflare.com/workers/wrangelr/webpack
https://github.com/cloudflare/wrangler/blob/master/docs/content/kv_commands.md => https://developers.cloudflare.com/workers/wrangelr/kv_commands
...
https://github.com/cloudflare/wrangler/blob/master/docs/content/environments.md => https://developers.cloudflare.com/workers/wrangelr/configuration/environments
```

File: README.md (hunk 3 above)
Comment (ashleymichal) [reply README.md]: they've been like that forever?

File: README.md (hunk 4 above)
Comment (ashleymichal) [reply README.md]: 🤦‍♀

**Pull request merged**